app/controllers/journal_entries_controller.rb

```diff
@@ -0,0 +1,50 @@
+class JournalEntriesController < ApplicationController
+
+  def index
+    @journal_entries = JournalEntry.all
+    redirect_to journals_path
```

Comment:
do we need this explicit redirection?

Reply:
Yes i think so, but i changed it so it fits the logic: 
(it should redirect to the journal, to display all the indexes)      

    @journal_entries = JournalEntry.all
    redirect_to journal_path(@journal)
  end